requirements.txt

```diff
@@ -16,4 +16,3 @@ yarl==1.1.0
 idna<2.9,>=2.5
 requests==2.25.1
 chardet<4.0
-packaging==20.9
```

Comment:
we don't make any version comparison anymore in octobot ?

Reply:
It's a dependency of Tentacle-Manager now. It'll be installed with OctoBot-Tentacles-Manager==2.4.11

Reply:
I'm wondering if we should freeze the version of `packaging` in octobot and have a range of versions in OctoBot-Tentacles-Manager, what do you think about it ? Considering it's required in both

Reply:
I don't understand why we should do that, but we can.

Reply:
Not that it's necessary but in my opinion it would be cleaner to have it in both repo since both are importing this package directly (and removing it OctoBot-Tentacles-Manager for whatever reason if we don't need it anymore won't break OctoBot without us noticing it in OctoBot-Tentacles-Manager CI)

Reply:
We should then define a requirement policy, we don't do this for aiohttp and others. I agree that it would be cleaner, but it will be harder to maintain.